### PR TITLE
new ampltools package

### PR DESCRIPTION
**Pull request by Sangu-Mbekelu**

### CODE REVIEW

**Comment by Sangu-Mbekelu** [comment]
@spackbot fix style

File: var/spack/repos/builtin/py-ampltools/package.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```
Comment (tldahlgren) [inline var/spack/repos/builtin/py-ampltools/package.py]:
```suggestion
# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
```

**Review by tldahlgren — CHANGES_REQUESTED** [review]
This file is under the wrong directory and needs to be moved to `var/spack/repos/builtin/packages/py-ampltools`.

Otherwise, I confirmed the sha256.

**Comment by tldahlgren** [comment]
Adding @adamjstewart since it is a `PythonPackage`.

File: var/spack/repos/builtin/py-ampltools/package.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+
+class PyAmpltools(PythonPackage):
+    """This package includes tools to use with AMPL and amplpy."""
+
+    homepage = "https://ampl.com/"
+    pypi = "ampltools/ampltools-0.4.6.tar.gz"
+
+    version("0.4.6", sha256="d54b399c1d78d02e3f4023aa2335b57832deb7d31cdefe4e219e4f2a2bb19a83")
+
```
Comment (adamjstewart) [inline var/spack/repos/builtin/py-ampltools/package.py]: Missing build dep on setuptools

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Review by adamjstewart — APPROVED** [review]

**Review by tldahlgren — APPROVED** [review]
Re-confirmed the sha256.

**Pull request merged**